### PR TITLE
[ARM CPU] Fix MVN6 decompose drop accuracy

### PR DESCRIPTION
**Pull request by allnes**
CSV-111880

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const std::vector<float> epsilonF = {
+        0.0001f
+};
+
+const std::vector<std::string> epsMode = {
+        "inside_sqrt",
+        "outside_sqrt"
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_power_mvn_decomposition_3D, Mvn6LayerTest,
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp:32]: Mvn6LayerTest is seemed to be the class from the shared_tests.
Here we should work only with the plugin's single_layer_tests
Also, I do not think we need to deal with MVN here at all.
I understand that this eltwise-power node is the result of MVN decomposition.
But for accuracy verification we can just create an instance for the eltwise-power test (even the file itself is called eltwise.cpp)

**Review by EgorDuplensky — CHANGES_REQUESTED** [review]

File: src/plugins/intel_cpu/tests/unit/ngraph_transformations/arm/mnv6_power_decomposition.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    {
+        ov::pass::Manager manager;
+        manager.register_pass<MVN6PowerDecomposition>();
+        manager.run_passes(m);
+    }
+
+    auto list_of_ops = m->get_ordered_ops();
+    ASSERT_STRCASEEQ(list_of_ops[0]->get_type_info().name, "Parameter");
+    ASSERT_STRCASEEQ(list_of_ops[1]->get_type_info().name, "Multiply");
+    ASSERT_STRCASEEQ(list_of_ops[2]->get_type_info().name, "Result");
```
[the remark this answers is not in the document]
Comment (allnes) [reply src/plugins/intel_cpu/tests/unit/ngraph_transformations/arm/mnv6_power_decomposition.cpp]: @EgorDuplensky done

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp (hunk 1 above)
Comment (allnes) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/eltwise.cpp]: @EgorDuplensky done

**Review by EgorDuplensky — APPROVED** [review]

**Review by dmitry-gorokhov — CHANGES_REQUESTED** [review]
ACL doesn't support negative base for power operation. Given that current solution will still work incorrectly for cases with power != 2.
The proposal is not to enable ACL for power operation, but use optimized C++ instead.

**Pull request closed without merging**